src/main.cpp

```diff
@@ -1662,6 +1662,16 @@ void reset_creature_max_levels(void)
     }
 }
 
+void reset_hand_rules(void)
+{
+    struct DungeonAdd* dungeonadd;
+    for (int i = 0; i < DUNGEONS_COUNT; i++)
+    {
+        dungeonadd = get_dungeonadd(i);
+        memset(dungeonadd->hand_rules, 0, sizeof(dungeonadd->hand_rules));
```

Comment:
I believe the memset thing you're doing here is mostly for structs that are still used in the dll, however the dungeonAdd struct is a keeperFx specific struct so most likely can reset it without memset, not 100% sure maybe other dev can confirm